api-reference/10 UI Widgets/dxDiagram/1 Configuration/customShapes/toolboxWidthToHeightRatio.md

```diff
@@ -4,7 +4,9 @@ type: Number
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Specifies an aspect ratio of the shape in the toolbox.
 
 ---
-<!-- Description goes here -->
\ No newline at end of file
+The size of a shape in the toolbox is determined automatically based on the toolbox width ([width](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#width) and the number of shapes per row ([shapeIconsPerRow](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#shapeIconsPerRow). 
```

Comment:
Just checking, but I wanted to make sure you're using the correct number of parenthesis in this sentence. Do you need an additional parenthesis after the `width` url and the `shapeIconsPerRow` url?

Reply:
I can rewrite it as follows:
```suggestion
The size of a shape in the toolbox is determined automatically based on the toolbox [width](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#width and the [number of shapes per row](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#shapeIconsPerRow. 
```

Reply:
Is my suggestion below correct:

```suggestion
The size of a shape in the toolbox is determined automatically based on the toolbox [width](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#width) and the [number of shapes per row](/Documentation/ApiReference/UI_Widgets/dxDiagram/Configuration/toolbox/#shapeIconsPerRow).
```